### DISCUSSION TITLE
How to increase the font size of CircleMarker's legend?

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#627] [Q&A] [answered]
The legend font size is really small when the list for circlemarker is long:

![image](https://github.com/opengeos/leafmap/assets/30388627/73883210-c889-4c26-af8a-861389fb3fc8)

I have tried to modify the `add_circle_markers_from_xy`:
```
            for p in popup:
                html = html + "<b>" + p + "</b>" + ": " + str(row[p]) + "<br>"
            html = "<h2> " + html + " </h2>"
            popup_html = widgets.HTML(html)
```

It seems `<h2>` doesn't work at all.

**Comment by giswqs** [comment, marked as answer]
Fixed in https://github.com/opengeos/leafmap/pull/628. Update the package using `leafmap.update_package()` and restart the kernel.
